Add SMSPool

smspool opened the pull request:
**Type of pull request:** product addition
**Related issues:** list the related issues here (if applicable)

---

Privacy policy: https://www.smspool.net/privacy-policy
Terms of service: https://www.smspool.net/tos

Provide more information about your pull request here. For help, see our contribution guide: http://privacyspy.org/contribute/

smspool committed:
Create smspool.toml

smspool committed:
Update smspool.toml

smspool committed:
Create smspool.png

smspool committed:
Update smspool.toml

doamatto commented on products/smspool.toml:
```diff
@@ -0,0 +1,58 @@
+name = "SMSPool"
+description = "Protect your identity today with our one-time use or rental phone numbers."
```
This makes it seem like Politiwatch/PrivacySpy is endorsing SMSPool or that it's a product internally. Can we get this reworded so that it's just a description for the product itself ? Take a look at other products or let me know if you need examples of such.

doamatto replied:
Fixed in 2855b7271cc0534780a0c5fbdce51003640134d3.

doamatto commented on products/smspool.toml:
```diff
@@ -0,0 +1,58 @@
+[rubric.security]
+value = "na"
+notes = [
+  "SMSPool does not store or require any personal data"
```
Quoting and scoring based on Article 6 would be ideal. Data is still stored and collected nevertheless.

doamatto replied:
I feel quoting and scoring based on Article 6 would still be an ideal solution.

doamatto commented on products/smspool.toml:
```diff
@@ -0,0 +1,58 @@
+citations = [ "Smspool.net makes use of the services of third party companies that are processing your data. We are committed to only engage with those companies that fully comply with the GDPR and CCP, maintain the highest standards of data protection and never sell data to other parties. We only make use of the services of Cloudfare for Analytics, CDN and DDoS protection, based out of the Unite ...
+
+[rubric.history]
+value = "last-modified"
```
If I recall correctly, this will fail tests when I approve them to run. Just add a quote that shows the "Most recent updated" line and it should be fine.

doamatto replied:
I meant with the date too, but that *technically* works haha.

doamatto commented on products/smspool.toml:
```diff
@@ -0,0 +1,58 @@
+value = "last-modified"
+
+[rubric.data-deletion]
+value = "yes-automated"
```
This doesn't explicitly tell us that it's an automated process like using a delete button in a settings menu. Either the policy needs to be remedied or it should be scored as `yes-contact`.

doamatto replied:
Fixed in 2855b7271cc0534780a0c5fbdce51003640134d3.

doamatto commented on products/smspool.toml:
```diff
@@ -0,0 +1,58 @@
+]
+
+[rubric.third-party-access]
+value = "no"
```
Access is shared with Cloudflare (critical based on "CDN and DDoS protection") and Stripe (critical because payments). If that's all the parties (unsure myself), the `yes-specified-critical` value would be ideal.

doamatto replied:
Fixed in 2855b7271cc0534780a0c5fbdce51003640134d3.

doamatto commented on products/smspool.toml:
```diff
@@ -0,0 +1,58 @@
+  "\"We collect and process the following personal data from visitors and subscribers. The purpose of this will be to register you as a User to our services: Username, and optionally email address. In addition, we may store the IP addresses from each visitor, which is only used for statistical purposes, and this will not be linked to your personal data.\""
+]
+
+[rubric.noncritical-purposes]
```
Will fail test. Needs a citation or note. I know @milesmcc was talking with me a while back about how some analytics might be deemed critical. I'll touch base with him off GH and see if he wants to move the line for analytics or not. Assume for the time being that won't be the case, which means analytics is non-critical.

doamatto replied:
The verdict here was that analytics will continue to be marked as non-critical. Since Cloudflare's analytics are mentioned in Article 2, this should be marked as `no` ; unless you mean the site statistics from having stuff under clouds and not their proper analyitcs offering, then it would stay as `na`. No policy change would be needed, just clarification here.

doamatto replied:
The verdict here was that analytics will continue to be marked as non-critical. Since Cloudflare's analytics are mentioned in Article 2, this should be marked as `no` ; unless you mean the site statistics from having stuff under clouds and not their proper analyitcs offering, then it would stay as `na`. No policy change would be needed, just clarification here.

doamatto replied:
Going to assume it's their analytics and not the site statistics. Will welcome the change if it's clarified later on.

doamatto commented on products/smspool.toml:
```diff
@@ -0,0 +1,58 @@
+value = "na"
+
+[rubric.law-enforcement]
+value = "reasonable"
```
This should be `unspecified` with a citation on the tracking for fradulent behaviour. There's no mention of when data would be shared with law enforcement.

doamatto replied:
Fixed in 2855b7271cc0534780a0c5fbdce51003640134d3.

doamatto commented on products/smspool.toml:
```diff
@@ -0,0 +1,58 @@
+value = "reasonable"
+
+[rubric.list-collected]
+value = "na"
```
Should be `summarily`. Missing note or citation.

doamatto replied:
Fixed in 2855b7271cc0534780a0c5fbdce51003640134d3.

doamatto commented on products/smspool.toml:
```diff
@@ -0,0 +1,58 @@
+value = "na"
+
+[rubric.revision-notify]
+value = "na"
```
This is a tough one that I'll also bring up to @milesmcc — you can for some users but not all. I think `na` works here, but I'll run it by him regardless just to get another pair of eyes on it. Also please add a blank line to the bottom of your file

doamatto replied:
The note works well here.

Fixed in 2855b7271cc0534780a0c5fbdce51003640134d3.

doamatto reviewed (changes requested):
Overall a good start. I'll approve checks and look it over once some changes are made.

smspool commented:
Cheers for the input, I will work on it now.

smspool committed:
Update smspool.toml

smspool committed:
Update CONTRIBUTORS.toml

doamatto reviewed (changes requested):
Sorry for the wait and thanks for your patience. Just a few more things (and making sure the tests pass) and then I can get this merged for you (no new comments, just add-ons to the ones above).

doamatto committed:
fix(product): updates to SMSPool

- Updated description to use more neutral wording
- Adjusted score for security
- Added CF to noncritical

doamatto committed:
Merge branch 'master' into master

doamatto commented:
Waiting for checks to pass, but will merge shortly after. Thanks !

smspool commented:
Not a problem, thank you so much.
